restcomm/restcomm.ussd/src/main/java/org/restcomm/connect/ussd/telephony/UssdCallManager.java

```diff
@@ -282,7 +282,7 @@ private ActorRef outbound(final Object message) throws ServletParseException {
         SipURI to = (SipURI)sipFactory.createSipURI(request.to(), uri);
 
         String transport = (to.getTransportParam() != null) ? to.getTransportParam() : "udp";
-        from = outboundInterface(transport);
+        //from = outboundInterface(transport);
```

Comment:
@croufay do you mean `outboundInterface ` is not used at all in `USSDCallManger`'s case?
if yes then would it make sense to remove this method? as it is private and this was only reference to this method?